Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "Unknown141", "type": "uint32" },
+        { "name": "Unknown142", "type": "uint32" },
+        { "name": "Unknown143", "type": "uint32" },
+        { "name": "Unknown144", "type": "uint32" },
+        { "name": "Unknown145", "type": "uint32" },
+        { "name": "Unknown146", "type": "uint32" },
+        { "name": "Unknown147", "type": "uint32" },
+        { "name": "Unknown148", "type": "uint32" },
+        { "name": "Unknown149", "type": "uint32" },
+        { "name": "Unknown150", "type": "uint32" },
+        { "name": "Unknown151", "type": "uint32" },
+        { "name": "Unknown152", "type": "uint32" },
+        { "name": "Unknown153", "type": "uint32" },
+        { "name": "Unknown154", "type": "uint32" },
+        { "name": "Unknown155", "type": "uint32" },
+        { "name": "Unknown156", "type": "uint32" },
+        { "name": "Unknown157", "type": "uint32" },
+        { "name": "Unknown158", "type": "uint32" },
+        { "name": "Unknown159", "type": "uint32" },
+        { "name": "Unknown160", "type": "uint32" },
+        { "name": "Unknown161", "type": "uint32" },
+        { "name": "Unknown162", "type": "uint32" },
+        { "name": "Unknown163", "type": "uint32" },
+        { "name": "Unknown164", "type": "uint32" },
+        { "name": "Unknown165", "type": "uint32" },
+        { "name": "Unknown166", "type": "uint32" },
+        { "name": "Unknown167", "type": "uint32" },
+        { "name": "Unknown168", "type": "uint32" },
+        { "name": "Unknown169", "type": "uint32" },
+        { "name": "Unknown170", "type": "uint32" },
+        { "name": "Unknown171", "type": "uint32" },
+        { "name": "Unknown172", "type": "uint32" },
+        { "name": "Unknown173", "type": "uint32" },
+        { "name": "Unknown174", "type": "uint32" },
+        { "name": "Unknown175", "type": "uint32" },
+        { "name": "Unknown176", "type": "uint32" },
+        { "name": "Unknown177", "type": "uint32" },
+        { "name": "Unknown178", "type": "uint32" },
+        { "name": "Unknown179", "type": "uint32" },
+        { "name": "Unknown180", "type": "uint32" },
+        { "name": "Unknown181", "type": "uint32" },
+        { "name": "Unknown182", "type": "uint32" },
+        { "name": "Unknown183", "type": "uint32" },
+        { "name": "Unknown184", "type": "uint32" },
+        { "name": "Unknown185", "type": "uint32" },
+        { "name": "Unknown186", "type": "uint32" },
+        { "name": "Unknown187", "type": "uint32" },
+        { "name": "Unknown188", "type": "uint32" },
+        { "name": "Unknown189", "type": "uint32" },
+        { "name": "Unknown190", "type": "uint32" },
+        { "name": "Unknown191", "type": "uint32" },
+        { "name": "Unknown192", "type": "uint32" },
+        { "name": "Unknown193", "type": "uint32" },
+        { "name": "Unknown194", "type": "uint32" },
+        { "name": "Unknown195", "type": "uint32" },
+        { "name": "Unknown196", "type": "float32" }
+      ]
+    },
```

Comment:
No parameter names were given because this is absolutely ridiculous what is this Sonic Team.